spec/std/string_spec.cr

```diff
@@ -1306,4 +1306,16 @@ describe "String" do
   it "gets length of \0 string" do
     "\0\0".length.should eq(2)
   end
+
+  context "%" do
+    it "substitutes one placeholder" do
+      res = "change %{this}" % { "this": "nothing" }
+      res.should eq "change nothing"
+    end
+
+    it "substitutes multiple placeholder" do
+      res = "change %{this} and %{more}" % { "this": "nothing", "more": "something" }
+      res.should eq "change nothing and something"
+    end
+  end
```

Comment:
I mean like `"%s %{foo}" % "bar"` and it should probably just ignore the substitutions like your current code does.